src/common/transformations/src/transformations/mlir/convert.cpp

```diff
@@ -606,8 +618,13 @@ mlir::OwningOpRef<mlir::ModuleOp> ngraph_to_mlir(MLIRContext* context,
         auto tensor = conversion_context.nodeOutputMap.at(outputs[i]);
         auto memref = func.getArgument(i + inputs.size());
         auto loc = createLocation(context, outputs[i].get_node_shared_ptr());
-        auto materialize = block_builder.create<bufferization::MaterializeInDestinationOp>(loc, tensor, memref);
-        materialize.setWritable(true);  // TODO: Can I set it as ctor argument above?
+        // TODO: Is restrict always guaranteed?
```

Comment:
Yes it is guaranteed at this stage because OpenVINO custom kernels mechanism that we are leveraging here doesn't support in-place semantics hence all output tensors don't have aliases in the scope of such simple program.

Reply:
That's great, removed to TODO and we can leave it marked as restrict.